config/index.js

```diff
@@ -493,6 +493,57 @@ var conf = convict({
       default: '.+@mozilla\\.com$',
       env: 'LASTACCESSTIME_UPDATES_EMAIL_ADDRESSES'
     }
+  },
+  signinUnblock: {
+    codeLength: {
+      doc: 'Number of base36 digits to make up an unblockCode',
+      default: 8,
+      env: 'SIGNIN_UNBLOCK_CODE_LENGTH'
+    },
+    codeLifetime: {
+      doc: 'How long an unblockCode should be valid for',
+      format: 'duration',
+      default: '1 hour',
+      env: 'SIGNIN_UNBLOCK_CODE_LIFETIME'
+    },
+    enabled: {
+      default: true
+    },
+    allowEmailRegex: {
+      doc: 'If feature enabled, allow sign-in unblock for email addresses matching this regex.',
+      format: Array,
+      default: [
+        '.+@mozilla\.com$'
+      ],
+      env: 'SIGNIN_UNBLOCK_ALLOW_EMAIL_REGEX'
+    },
+    forceEmailRegex: {
```

Comment:
I'm also not a massive fan of the ad-hoc naming convention we've developed for these regexes. I think they can be tweaked to make the code that uses them a bit clearer, and I'm also not a fan of the almost-hungarian-notation `...Regex` suffix.

Feel free to ignore if I'm bike-shedding, but I think something along the lines of `forcedEmailAddresses` would be clearer for this one and `enabledEmailAddresses` would be clearer for the one above.